raven/src/main/webapp/WEB-INF/web.xml

```diff
@@ -47,6 +47,16 @@
         <load-on-startup>2</load-on-startup>
     </servlet>
 
+  <servlet>
+    <servlet-name>PubKeyServlet</servlet-name>
+    <servlet-class>ca.nrc.cadc.rest.RestServlet</servlet-class>
+    <init-param>
+      <param-name>get</param-name>
+      <param-value>org.opencadc.raven.GetPubKeyAction</param-value>
```

Comment:
I don't see this class included in your PR... but I think we can re-use the one from vault after moving it to cadc-inventory-server. So don't fix right now... I'll try to integrate

Reply:
Sorry - forgot to add the file to the commit but I have it locally if you need it.